packages/@lwc/errors/src/compiler/error-info/lwc-class.ts

```diff
@@ -218,4 +218,4 @@ export const DecoratorErrors = {
         level: DiagnosticLevel.Error,
         url: '',
     },
-};
+} as const;
```

Comment:
Allows typing the message placeholders (more details in another comment). Hopefully this is ok.